src/components/SocialMediaLinks/SocialMediaLinks.test.tsx

```diff
@@ -0,0 +1,474 @@
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with labels set to true", async () => {
+      const { container } = render(<SocialMediaLinks showLabels={true} />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+  describe("Colors", () => {
+    it("passes axe accessibility test with link color & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} color={"link"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with textInverse color & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} color={"textInverse"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+  describe("Borders", () => {
+    it("passes axe accessibility test with straight borders & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} border={"straight"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with circular borders", async () => {
+      const { container } = render(<SocialMediaLinks border={"circular"} />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+});
+
+describe("SocialMediaLinks", () => {
+  it("Renders default", () => {
+    render(<SocialMediaLinks />);
+
+    const links = screen.getAllByRole("link");
+
+    // All the types in SocialMediaDataMap.ts should be there.
+    expect(links).toHaveLength(9);
+
+    // Spot tests.
+
+    // Links.
+    expect(links[1]).toHaveAttribute("href", "facebook.com/nypl");
+    expect(links[4]).toHaveAttribute("href", "soundcloud.com/nypl");
+    expect(links[6]).toHaveAttribute("href", "nypl.tumblr.com");
+
+    // Labels.
+    expect(screen.getByText("NYPL Blog")).toBeInTheDocument();
+    expect(screen.getByText("Pinterest")).toBeInTheDocument();
+    expect(screen.getByText("TikTok")).toBeInTheDocument();
+
+    // Icons.
+    expect(screen.getByTitle("socialInstagram icon")).toBeInTheDocument();
+    expect(screen.getByTitle("socialTwitter icon")).toBeInTheDocument();
+    expect(screen.getByTitle("socialYoutube icon")).toBeInTheDocument();
+  });
+
+  it("Renders selected types", () => {
+    render(
+      <SocialMediaLinks
+        layout={"column"}
+        linksData={[
+          {
+            labelText: "Alt Twitter",
+            type: "twitter",
+            url: "twitter.com/elsewhere",
+          },
+          {
+            labelText: "NYPL Facebook",
+            type: "facebook",
+            url: "facebook.com/nypl",
+          },
+        ]}
+      />
+    );
+
+    // @todo Do we need to test if the icons are in a row or column?
+    // @todo Or if the colors are correct?
+    // @todo Or if the borders are correct?
```

Comment:
The snapshots are enough! Those are generally harder to test in unit tests anway.